images/build/Dockerfile

```diff
@@ -1,3 +1,5 @@
 FROM registry.access.redhat.com/ubi8/go-toolset:1.14.12
+MAINTAINER CodeReady Containers <devtools-cdk@redhat.com>
+
```

Comment:
same for this, could be squashed in initial commit?

Reply:
does this prevent approval?

Reply:
I'd prefer to have these squashed, but not going to fight for it. At the moment it makes the history messy

```
Move container to images folder

+ MAINTAINER CodeReady Containers <devtools-cdk@redhat.com>
```
does not make a lot of sense.